.github/workflows/nightly_clang_build.yml

```
@@ -1,33 +1,31 @@
 name: Clang nightly build
 
 on:
-  workflow_dispatch:
   schedule:
     - cron: "0 19 * * *" # run at 4 AM JST
+  workflow_dispatch:
 
 jobs:
   build-using-clang:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     container: ubuntu:focal
 
     steps:
       - name: Check out repo
-        uses: actions/checkout@v2.3.4
+        uses: actions/checkout@v2
 
       - name: Setup ROS environment
-        uses: ros-tooling/setup-ros@0.1.1
+        uses: ros-tooling/setup-ros@v0.1
         with:
           required-ros-distributions: foxy
 
       - name: Run action-ros-ci
         id: action_ros_ci_step
-        uses: ros-tooling/action-ros-ci@0.1.2
+        uses: ros-tooling/action-ros-ci@v0.2
         with:
           package-name: obstacle_stop_planner_refine
           target-ros2-distro: foxy
           vcs-repo-file-url: build_depends.repos
           colcon-mixin-name: clang-libcxx
```

I found some packages cannot build with libc++, so we have to pass the appropriate build options manually instead of `clang-libcxx`.
```suggestion
          colcon-defaults: |
            {
              "build": {
                "mixin": ["release", "compile-commands"]
              }
            }
```